test/extended/router/h2spec.go

```diff
@@ -84,6 +84,9 @@ var _ = g.Describe("[sig-network-edge][Conformance][Area:Networking][Feature:Rou
 			canaryImage, err := getCanaryImage(oc)
 			o.Expect(err).NotTo(o.HaveOccurred())
 
+			err = oc.AsAdmin().Run("adm", "policy", "add-scc-to-user").Args("restricted", "-n", oc.Namespace(), "-z", "default").Execute()
```

Comment:
how is this cleaned up at the end of the e2e run?

Reply:
> how is this cleaned up at the end of the e2e run?

How do you make sure the kube-apiserver observes this before creating the pod.

Reply:
> how is this cleaned up at the end of the e2e run?
it's a rolebinding, it gets removed on NS removal

> How do you make sure the kube-apiserver observes this before creating the pod.
currently, we don't